8-algoruthm-and-data-structures-2/hometask/src/JobRunner.ts

```diff
@@ -0,0 +1,79 @@
+interface Job {
```

Comment:
I'd have several interfaces, for instance Job and PriorityJob (will each job have priority?).